src/libraries/Directory.Build.props

```diff
@@ -1,15 +1,12 @@
-<Project TreatAsLocalProperty="TargetOS">
```

Comment:
Unrelated: Removing the TreatAsLocalProperty attribute which isn't used anymore.